tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -60,6 +60,10 @@ def _run(self):
                     SyncObject(data=json.dumps(raw_data))
                 sync_object.save()
 
+                gluster_brick_dir = NS.gluster.objects.\
+                                     GlusterBrickDir()
+                gluster_brick_dir.save()
```

Comment:
The value of the default brick directory should be set just once, during the cluster import. That way, it would be applicable to both the import and create scenarios. The value would not need to be synced ever again. Essentially, it becomes read-only. If needed, in the future, any additional brick directories (manually added by the user or based on detection from the initial cluster import) could be added to a separate list object.